doc/sphinx/addendum/micromega.rst

```diff
@@ -54,6 +54,10 @@ or only for reals by ``Require Import Lra``.
 
    This :term:`flag` (set by default) instructs :tacn:`nra` to cache its results in the file `.nra.cache`
 
+.. flag:: Psatz Conversion
+
+   This :term:`flag` (unset by default) instructs the tactics :tacn:`lia`, :tacn:`nia`, :tacn:`lra`, :tacn:`nra`
+   and :tacn:`psatz` to use the convertibility test (instead of syntactic equality) to detect equal terms during syntaxfication.
```

Comment:
```suggestion
   When set, use :term:`definitional equality` (:term:`convertibility <convertible>`) instead of
   syntactic equality to detect equal terms in the :tacn:`lia`, :tacn:`nia`, :tacn:`lra`,
   :tacn:`nra` and :tacn:`psatz` tactics during syntaxification.  The default is unset.
```

Again, what is syntaxification?